app/constants/view.js

```diff
@@ -109,10 +109,10 @@ const ViewTypes = keyMirror({
 });
 
 const RequiredServer = {
-    FULL_VERSION: 5.19,
+    FULL_VERSION: '5.25.5',
     MAJOR_VERSION: 5,
-    MIN_VERSION: 19,
-    PATCH_VERSION: 0,
+    MIN_VERSION: 25,
+    PATCH_VERSION: 5,
```

Comment:
I don't think we need the patch version, maybe @amyblais and @esethna can clarify on that

Reply:
I think just "5.25" is fine, but Eric can confirm.

Reply:
Agree @migbot ^